### PR TITLE
Activations

### PR DESCRIPTION
**Pull request by vishalmanes109**
Identity and BinaryStep activation function implemented.
change the formula of the already implemented linear activation function. 
also implemented cross_entropy loss function with and without kl_divergence 

Readme updated and docstring added.

### CODE REVIEW

**Review by RyanDsilva — APPROVED** [review]
Looks good!
